docs/en/glossary/glossary.asciidoc

```diff
@@ -512,13 +527,27 @@ bucket in an {anomaly-job}. For more information, see
 {ml-docs}/ml-influencers.html[Influencers].
 //Source: X-Pack
 
+[[glossary-ingestion]] ingestion::
+The process of collecting and sending data from various data sources to {es}.
+//Source: Observability
+
 [[glossary-input-plugin]] input plugin::
 A {ls} <<glossary-plugin,plugin>> that reads <<glossary-event,event>> data
 from a specific source. Input plugins are the first stage in the {ls}
 event processing <<glossary-pipeline,pipeline>>. Popular input plugins include
 file, syslog, redis, and beats.
 //Source: Logstash
 
+[[glossary-instrumentation]] instrumentation::
+Extending application code to track where your application is spending time.
+Code is considered instrumented when it collects and reports this performance data to APM.
+//Source: Observability
+
+[[glossary-integration]] integration::
+Out-of-the-box configurations for common data sources to simplify the collection,
```

Comment:
I'm not sure we can solve this problem in the glossary, but users need to understand that the integrations available for Fleet (fka Integration Manager) are different from the integrations provided by modules. WDTY?